src/index.html

```diff
@@ -3,12 +3,76 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+    crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Moyo header</h1>
+
+  <body id="home">
+
+    <header class="header">
+      <a href="#home" class="logo">
+        <img
+        src="./images/logo.png"
+        alt="logo-img.png"
+        class="logo__img">
```

Comment:
```suggestion
        <img
          src="./images/logo.png"
          alt="logo-img.png"
          class="logo__img"
        >
```
Fix all cases
![image](https://github.com/mate-academy/layout_moyo-header/assets/96409563/cad6145e-0290-4b74-a236-1c840738a3d9)